pytensor/tensor/basic.py

```diff
@@ -255,7 +255,26 @@ def _obj_is_wrappable_as_tensor(x):
 )
 
 
-def get_scalar_constant_value(
+def get_scalar_constant(v, elemwise=True, only_process_constants=False, max_recur=10):
```

Comment:
We shouldn't change the name, even if it's a bit verbose.

```suggestion
def get_scalar_constant_value(v, elemwise=True, only_process_constants=False, max_recur=10):
```